### PR TITLE
Added GitHub Actions to automate packaging

### PR DESCRIPTION
**Pull request by penev92**
Adding 3 workflows to automate compiling of native dependencies, compiling of the C# wrapper library and packaging it all into a NuGet package and publishing that.

The native binary compilation steps are all taken from @pchote's https://github.com/pchote/native-dependencies

P.S.: For results of a successful run see https://github.com/penev92/Eluant/actions/runs/4652147553 and https://www.nuget.org/packages/OpenRA-Eluant/1.0.21-a4

### CODE REVIEW

File: OpenRA-Eluant.nuspec [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<package>
+  <metadata>
+    <id>OpenRA-Eluant</id>
+    <version>1.0.21-a4</version>
+    <authors>OpenRA team</authors>
+    <owners>OpenRA team</owners>
```
Comment (Mailaender) [inline OpenRA-Eluant.nuspec]:
```suggestion
    <authors>OpenRA</authors>
    <owners>OpenRA</owners>
```

File: OpenRA-Eluant.nuspec [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <projectUrl>https://github.com/OpenRA/Eluant</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Eluant modified for OpenRA</description> <!-- TODO: -->
+    <copyright>Copyright 2019</copyright> <!-- TODO: -->
```
Comment (Mailaender) [inline OpenRA-Eluant.nuspec]:
```suggestion
    <copyright>Copyright (c) The OpenRA Developers and Contributors</copyright>
```

File: OpenRA-Eluant.nuspec [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    <!-- <license type="file">LICENSE.md</license>  --> <!-- TODO: -->
```
Comment (Mailaender) [inline OpenRA-Eluant.nuspec]:
```suggestion
    <license type="expression">MIT</license>
```

File: OpenRA-Eluant.nuspec [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+    <!-- TODO: README -->
+    <tags>OpenRA Lua scripting</tags> <!-- TODO: -->
```
Comment (Mailaender) [inline OpenRA-Eluant.nuspec]: Remove the TODOs. There is really nothing to gain by optimizing these.

**Comment by penev92** [comment]
Done, thanks.

**Pull request merged**